cfpb/consumerfinance.gov
# Wagtail charts: Tweak code and add to learn pages

## Changes

- Auto lint things.
- Make ems rems.
- Add dedicated color variables.
- Add to learn pages.


## How to test this PR

1. `python cfgov/manage.py migrate` and `yarn build` and run a local server and you should be able to add a chart to the content area of a learn page.
